src/canvas/analyze-canvas-data-py/readme.md

```diff
@@ -5,17 +5,19 @@
 Input:
 CSV files with information about:
 
-* Courses
-* Assignments
-* Students
-* Enrollments
-* Submissions
+-   Courses
+-   Assignments
+-   Students
+-   Enrollments
+-   Submissions
```

Comment:
I think `*` is more standard than `-` for bullets.

Reply:
Fixed